modules/007-registrypackages/images/containerd-fe/werf.inc.yaml

```diff
@@ -1,48 +0,0 @@
-{{- $version := "1.5.11" }}
-{{- $image_version := $version | replace "." "-" }}
-
----
-image: {{ $.ModuleName }}/{{ $.ImageName }}-{{ $image_version }}
-from: {{ $.Images.BASE_SCRATCH }}
-import:
-- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-{{ $image_version }}
-  add: /
-  to: /
-  includePaths:
-  - containerd
-  - install
-  - uninstall
-  before: setup
-docker:
-  LABEL:
-    distro: all
-    version: all
-    containerd: {{ $version }}
----
-artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-{{ $image_version }}
-from: {{ $.Images.BASE_GOLANG_16_BUSTER }}
-git:
-- add: /modules/007-{{ $.ModuleName }}/images/{{ $.ImageName }}/scripts
-  to: /
-  stageDependencies:
-    setup:
-    - '**/*'
-mount:
-- fromPath: ~/go-pkg-cache
-  to: /go/pkg
-shell:
-  beforeInstall:
-  - apt-get update
-  - apt-get install -y patch libbtrfs-dev
-  install:
-  - mkdir -p /src
-  - cd /src
-  - git clone https://github.com/containerd/containerd.git
-  - cd containerd
-  - git checkout tags/v{{ $version }}
-  - git config --global user.email "builder@deckhouse.io"
-  - git cherry-pick 5f3ce9512b74718ac9519196ba70dfb1ee075fbe # Do not append []string{""} to command to preserve Docker and Werf compatibility
```

Comment:
This is still not in release of containerd - https://github.com/containerd/containerd/blob/release/1.6/pkg/cri/opts/spec.go#L66

Reply:
It seem that we should wait for https://github.com/containerd/containerd/pull/7298 to be merged.